tls/s2n_connection.c

```diff
@@ -538,7 +533,6 @@ int s2n_connection_wipe(struct s2n_connection *conn)
     /* Some required structures might have been freed to conserve memory between handshakes.
      * Restore them.
      */
-
```

Comment:
Nit: If you're going to remove this line, you'll also need to remove the line on line 547/541. The comment also applies to the prf_space.